content/tor-browser-for-android/content.lr

```diff
@@ -0,0 +1,78 @@
+_model: topic
+---
+title: Tor Browser for Android
+---
+description: Learn about Tor Browser for Android and how to install it
+---
+seo_slug: tor-broser-for-android
+---
+body:
+
+##### INTRODUCTION - A LITTLE BIT OF HISTORY
+
+[Mobile browsing](https://www.theguardian.com/technology/2016/nov/02/mobile-web-browsing-desktop-smartphones-tablets) 
+has increasingly become the primary way many people go online.
```

Comment:
Please write one sentence per line, as the translation plugin will break this paragraph in different strings and will make it hard to translate. i.e. 
```
[Mobile browsing](https://www.theguardian.com/technology/2016/nov/02/mobile-web-browsing-desktop-smartphones-tablets) has increasingly become the primary way many people go online.
```
instead of:
```
[Mobile browsing](https://www.theguardian.com/technology/2016/nov/02/mobile-web-browsing-desktop-smartphones-tablets) 
has increasingly become the primary way many people go online.
```

And the same in the rest of the text.

Reply:
Got it